web/src/views/SubmissionPortal/Components/SubmissionList.vue

```diff
@@ -226,8 +226,8 @@ export default defineComponent({
           <template #[`item.templates`]="{ item }">
             {{ item.metadata_submission.templates.map((template) => HARMONIZER_TEMPLATES[template].displayName).join(' + ') }}
           </template>
-          <template #[`item.created`]="{ item }">
-            {{ new Date(item.created).toLocaleString() }}
+          <template #[`item.date_last_modified`]="{ item }">
+            {{ new Date(item.date_last_modified).toLocaleString() }}
```

Comment:
I can't believe I've missed this bug before. We store all of our datetime as UTC in the database (which is good). But when they're serialized back out to JSON via Pydantic, it doesn't know what timezone they're in, so they come out as timezone-less strings like `"2024-12-17T01:20:58.801132"`. That means the we end up displaying the UTC time on the frontend.

You can see this now on dev and prod. If you create a new submission and then immediately go back to the submission portal home page you'll see the Created column for you new study reads 8 hours from now (assuming you're in Pacific Standard Time). 

The most comprehensive fix would probably be to change our Pydantic models so they serialize datetimes as either UNIX timestamps or strings with timezone info so it's less ambiguous. Alternatively, a short-term fix would be to append the letter `Z` (indicating UTC) to the `date_last_modified` value as it's passed to the `Date` constructor.
```suggestion
            {{ new Date(item.date_last_modified + 'Z').toLocaleString() }}
```

Reply:
I had noticed that the time zone was off when doing testing for this, but I had just assumed that was just something funky due to running a local server on Docker, but didn't notice it on prod! I'll add the short term fix in for now